add release override instruction, gcp create/teardown scripts

sallyom opened the pull request:
* Add a teardown-gcp script
* Increase to 2 workers
* update README, move images info to IMAGES.md
* For successful install, you have to override the release image.  Otherwise see errors in bootstrap node like:
```
Error reading manifest 4.5 in registry.svc.ci.openshift.org/origin/release: manifest unknown: manifest unknown
```

sallyom committed:
add release override instruction, add README instructions for teardown, teardown script

sallyom commented:
@ironcladlou ptal thanks :)

ironcladlou commented on README.md:
```diff
@@ -1,54 +1,56 @@
-  --metadata-from-file openshift-pull-secret=openshift-pull-secret.json
+$ gcloud beta compute ssh --zone "us-east1-c" $CLUSTER_NAME --project "openshift-gce-devel"
+$$ export OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE=registry.svc.ci.openshift.org/ocp/release:whatever
+$$ create-cluster nested
```
double $'s

One-liner alternative:
```
OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE=registry.svc.ci.openshift.org/ocp/release:whatever create-cluster nested
```

sallyom replied:
fixed

ironcladlou commented on README.md:
````diff
@@ -60,46 +62,18 @@ Tools can be updated right from the instance itself.
 ```shell
-$ update-installer
+$ gcloud beta compute ssh --zone "us-east1-c" $CLUSTER_NAME --project "openshift-gce-devel"
+$$ update-installer
````
Double $

sallyom replied:
fixed

ironcladlou commented on README.md:
````diff
@@ -1,54 +1,56 @@
 ```shell
-$ create-cluster nested
+$ gcloud beta compute ssh --zone "us-east1-c" $CLUSTER_NAME --project "openshift-gce-devel"
+$$ openshift-install destroy cluster --dir ~/clusters/$CLUSTER_NAME && rm -rf ~/clusters/$CLUSTER_NAME
````
Double $

sallyom replied:
fixed

ironcladlou commented on README.md:
````diff
@@ -60,46 +62,18 @@ Tools can be updated right from the instance itself.
-```shell
-$ packer build -var 'project=your-google-project-id' openshift4-libvirt.json
+$ gcloud beta compute ssh --zone "us-east1-c" $CLUSTER_NAME --project "openshift-gce-devel"
+$$ update-installer repo-owner branch
````
Double $

sallyom replied:
fixed

ironcladlou reviewed (commented):
Just a few minor docs issues

sallyom commented on tools/create-cluster:
```diff
@@ -53,7 +54,7 @@ platform:
       if: tt0
 publish: External
-pullSecret: '$(curl http://metadata.google.internal/computeMetadata/v1/instance/attributes/openshift-pull-secret -H "Metadata-Flavor: Google")'
+pullSecret: $(echo \'"${PULL_SECRET}"\')
```
@crawford here, this is the change that is already in the image in openshift-gce-devel-ci - I updated it to test my release PR, too early-but didn't think it would be an issue bc the libvirt template/setup is broken as/is anyways.

crawford replied:
Thanks! That makes much more sense now.

nit: `”${PULL_SECRET}”` (no need for the subshell)

sallyom commented:
@ironcladlou can you merge this? notice I changed the pull-secret to be scp-d rather than passed as gce-metadata.  That's because w/ metadata, there's no way _not_ to print the pull-secret in the gce console - I tried - seems there's no way to hide that.  I also added the create/teardown scripts but if you don't approve of those, I'll take them out and edit the README accordingly.  Thanks! A few projects depend on this setup, right now it's way broken for them.

crawford commented on tools/create-cluster:
```diff
@@ -20,6 +20,7 @@ fi
 export BASE_DOMAIN=openshift.testing
 export CLUSTER_NAME="${NAME}"
 export PUB_SSH_KEY="${SSH_KEY}.pub"
+PULL_SECRET=$(cat "${HOME}/pull-secret")
```
nit: `$(< “${HOME}/pull-secret”)`
